.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,11 +2,5 @@
 
 **JIRA:** Link to JIRA ticket
 
-**Merge checklist:**
-- [ ] All reviewers approved
-- [ ] CI build is green
-- [ ] Commits are squashed
```

Comment:
> Commits are squashed >> Let's remove 'Merge' and force 'Squash and Merge'

We could update this one, but I don't think we want to force 'Squash and Merge', because sometimes multiple commits makes sense.  Could be something like?

```
- [ ] Commits are squashed as desired (unless using Squash and Merge)
```